src/macos/ScrollView.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * ScrollView.tsx
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT license.
+ *
+ * RN Mac-specific implementation of the cross-platform ScrollView abstraction.
+ */
+
+import * as React from 'react';
+import * as RN from 'react-native';
+
+import EventHelpers from '../native-common/utils/EventHelpers';
+import { ScrollView as ScrollViewBase } from '../native-common/ScrollView';
+
+export class ScrollView extends ScrollViewBase {
+    protected _render(nativeProps: RN.ScrollViewProps & React.Props<RN.ScrollView>): JSX.Element {
+        const onKeyDownCallback = this.props.onKeyPress ? this._onKeyDown : undefined;
+
+        // Have to hack the windows-specific onKeyDown into the props here.
```

Comment:
This comment is misleading. I'm guessing that you copied it from the Windows UWP code?

Reply:
You're right :) There's one less prop than windows, so it can't be shared in a common file. Updated